layouts/partials/home/hero.html

```diff
@@ -17,7 +17,7 @@
       <br />
 
       <div class="buttons is-centered">
-        <a class="button is-primary is-inverted is-medium is-radiusless" href="/docs/current/">
+        <a class="button is-primary is-inverted is-medium is-radiusless" href="/docs/latest/">
```

Comment:
I'm not happy that this breaks the button on `localhost`, even when using `netlify dev` this doesn't forward correctly (it brings you to http://localhost:8888/docs/), but it does work on the deploy preview/will work on the production build.

Reply:
Hmm, it works for me.

Reply:
Oh, maybe not ... wait.

Reply:
I can confirm that it works for me locally using `netlify dev`

Reply:
I verify your confirmation. Thanks!

Reply:
No, I take back my verification -- the frontpage button on `localhost:8888` takes me to http://localhost:8888/docs/, not http://localhost:8888/docs/v3.4/